### PR TITLE
Feature/toxic log removal

### PR DESCRIPTION
**Pull request by pepisg**
In this PR I removed the toxic logs from staging and production applications
` No stream match for pointcloud chosen texture Process - Color`

### CODE REVIEW

**Review by charlielito — CHANGES_REQUESTED** [review]
I don't like this logic inside this code.  For example if the applications names change this brakes.
This should be an env vars for just enablind the logging, and in Kronos we set that env variable depending the application

**Comment by pepisg** [comment]
What do you think about this new implementation @charlielito ?

File: realsense2_camera/include/base_realsense_node.h [hunk 1]
```diff
@@ -388,6 +388,9 @@ namespace realsense2_camera
         double getImuPitch();
         void ChassisTransformTmrCb();
 
+        //Toxic logs removal
+        bool _display_logs=true;
```
Comment (charlielito) [inline realsense2_camera/include/base_realsense_node.h]: maybe a better name like `_texture_display_logs`

**Review by charlielito — CHANGES_REQUESTED** [review]
This looks good. Nevertheless I'll recommend to add this new env var as a parameter of the node, like the other parameters that this node already has, so you don't need to implement that `getEnv` function and this could just be passed in the launch file like the other parameters

File: realsense2_camera/src/base_realsense_node.cpp [hunk 2]
```diff
@@ -1072,10 +1053,14 @@ void BaseRealSenseNode::getParameters()
...
     // Kiwi added
     setNgetNodeParameter(_color_virtual_cam, "color_virtual_cam", COLOR_VIRTUAL_CAMERA);
+    setNgetNodeParameter(_texture_display_logs, "texture_display_logs", TEXTURE_DISPLAY_LOGS);
     setNgetNodeParameter(_robot_base_frame, "robot_base_frame", ROBOT_BASE_FRAME);
     setNgetNodeParameter(_camera_link_x, "camera_link_x", CAMERA_LINK_X);
     setNgetNodeParameter(_camera_link_y, "camera_link_y", CAMERA_LINK_Y);
     setNgetNodeParameter(_camera_link_z, "camera_link_z", CAMERA_LINK_Z);
+
+    ROS_INFO_STREAM("Texture logs displaying: " << _texture_display_logs);
+    ROS_INFO_STREAM("Texture logs displaying: " << _camera_link_z);
```
Comment (charlielito) [inline realsense2_camera/src/base_realsense_node.cpp]: ?

**Review by charlielito — APPROVED** [review]
Remove that extra ROS_INFO_STREAM

**Pull request merged**